src/main/java/org/kuali/kfs/fp/document/web/struts/CapitalAssetInformationActionBase.java

```diff
@@ -73,7 +73,7 @@
      * selected asset numbers, the system control amount is redistributed equally among the assets
      * when the distribution method is "distribute cost equally".
      *
-     * @see org.kuali.kfs.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
+     * @see org.kuali.rice.kns.web.struts.action.KualiAction#refresh(org.apache.struts.action.ActionMapping,
```

Comment:
This is just a reference in the javadoc comment on a method. An upgrade script I ran changed the package in our overlay file from rice to kfs, but I was changing it back to be the same as what is in the base code.